.gitignore

```diff
@@ -64,3 +64,4 @@ cloudshell_config.yml
 shells/generic_terraform_service/tests/.env
 tests/.env
 shells/backends/azure_tf_backend/tests/.env
+package/tests/integration_tests/int_tests_secrets.env
```

Comment:
instead of adding specific files with full paths it's better to add ".env" and it should filter out all files.